crates/noirc_evaluator/src/ssa_refactor/ir/dfg.rs

```diff
@@ -103,6 +105,8 @@ impl DataFlowGraph {
     /// block's id.
     ///
     /// The pairs are order by id, which is not guaranteed to be meaningful.
+    /// TODO: do we need to order by reverse post order? Is this useful? Check brillig
+    /// TODO: ie do we need to get all successors/predecessors first?
```

Comment:
This function is just for when a pass or test needs to iterate through each block and doesn't particularly care about the order (or whether the blocks are even still reachable). For passes that do care about the order, they can create a `PostOrder` directly like the Dead Instruction Elimination pass does.